.repositories.yaml

```diff
@@ -40,19 +40,6 @@ repositories:
        url: https://github.com/eclipse/che/tree/master/tests
        description: >-
          source code of our integration tests.
-  - url: https://github.com/eclipse-che/che-theia
-    name: che-theia
-    description: >-
-      Theia IDE integrated in Che.
-    devfile: https://github.com/eclipse-che/che-theia/blob/main/devfiles/che-theia-all.devfile.yaml
-    doc: https://github.com/eclipse-che/che-theia/blob/main/CONTRIBUTING.md
-    useCheReleaseLifecycle: true
-    components:
-     - name: generator
-       url: https://github.com/eclipse-che/che-theia/tree/main/generator
-       description: >-
-         `che:theia init` CLI to prepare and build che-theia
-       doc: https://github.com/eclipse-che/che-theia/blob/main/generator/CONTRIBUTING.md
```

Comment:
Can we have the che-code section instead?

Reply:
Already in the file... https://github.com/eclipse/che/blob/ba16c630f40fcb94cd8c69bc286db9dc41007768/.repositories.yaml#L50-L56